### DISCUSSION TITLE
How to build for windows 64?

### DISCUSSION THREAD

**Discussion by Kalai111** [variar/klogg#415] [Q&A]
I tried building for windows but failed because of incompatible QT library.
Do you have any docker environment with all the dependencies?

**Comment by variar** [comment]
Hi, docker containers are available for Ubuntu and Centos builds. On Windows it should be relatively easy to install required Qt version and point cmake at it. There is some build instructions in [BUILD.md](https://github.com/variar/klogg/blob/master/BUILD.md).
